src/components/Layout.js

```diff
@@ -71,22 +63,6 @@ const Layout = (props) => {
     } else {
       setIsDarkTheme(window.matchMedia("(prefers-color-scheme: dark)").matches)
     }
-    window
```

Comment:
Agree that these can be removed.

They DO add functionality (they listen to if the browser preferences are changed) but I think that's such a rare edge case to adjust for that we should just remove this.